README.md

```diff
@@ -6,7 +6,7 @@ OpenRefine is a power tool that allows you to load data, understand it,
 clean it up, reconcile it, and augment it with data coming from
 the web. All with the comfort and privacy of your own computer.
 
-[<img src="https://github.com/OpenRefine/OpenRefine/blob/master/graphics/icon/open-refine-256px.png" align="right">](http://openrefine.org)
+[<img src="./graphics/icon/open-refine-256px.png" style="width: 320px;" align="right">](http://openrefine.org)
```

Comment:
I think we need tokeep the full expanded URL, right ?  That way a local MD file reader and Github views will work just fine.

Reply:
Oh I see, once I verify that the logo is working I will switch it back to an absolute link then :)